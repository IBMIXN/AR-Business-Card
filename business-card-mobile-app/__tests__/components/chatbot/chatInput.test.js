import React from "react";
import renderer from "react-test-renderer";
import ChatMicInput from "../../../components/chatbot/chatInput";

describe("<ChatMicInput recording={false}/>", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ChatMicInput recording={false} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe("<ChatMicInput recording={true}/>", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ChatMicInput recording={true} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe("<ChatMicInput loading={true}/>", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ChatMicInput loading={true} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
