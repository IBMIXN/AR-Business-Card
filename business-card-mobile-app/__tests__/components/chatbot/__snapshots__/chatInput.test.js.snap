// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChatMicInput loading={true}/> renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "justifyContent": "center",
      "padding": 3,
    }
  }
>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "white",
          "borderRadius": 50,
          "padding": 20,
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="black"
        hidesWhenStopped={true}
        size="large"
      />
    </View>
  </View>
</View>
`;

exports[`<ChatMicInput recording={false}/> renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "justifyContent": "center",
      "padding": 3,
    }
  }
>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID="mic-button"
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "white",
          "borderRadius": 50,
          "padding": 20,
        }
      }
    >
      <Text />
    </View>
  </View>
</View>
`;

exports[`<ChatMicInput recording={true}/> renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "justifyContent": "center",
      "padding": 3,
    }
  }
>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID="mic-button"
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "white",
          "borderRadius": 50,
          "padding": 20,
        }
      }
    >
      <Text />
    </View>
  </View>
</View>
`;
